Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/DemistoRESTAPI.js

```diff
@@ -15,13 +15,16 @@ sendMultipart = function (uri, entryID, body) {
         // do nothing, use the body as is in the request.
         logDebug('could not parse body as a JSON object, passing as is. body: ' + JSON.stringify(body));
     }
-
+    var key = [params.apikey? params.apikey : (params.api_key? params.api_key.password : '')];
```

Comment:
Why do we do it this way and not like this?
```suggestion
    var key = [params.apikey? params.apikey :  params.api_key.password ];
```
Existing users will have `params.apikey`. New users will have `params.api_key`, such that doing `params.api_key.password` will return the API key or 'undefined', but will not error.

Reply:
I tried this, but if no api-key is provided then the user will get the following error:
'TypeError: Cannot read property 'password' of undefined at script'

Reply:
How could this be? Doesn't `api_key` always exist? even if  `password`  was not filled?

Reply:
It exist, but if password isn't filled then api_key will hold null value and therefore trying to access .password will raise an error.

Reply:
Roger